vm/actor_interface/src/builtin/power/mod.rs

```diff
@@ -16,12 +16,18 @@ use serde::{Deserialize, Serialize};
 use anyhow::Context;
 
 /// Power actor address.
-/// TODO: Select based on actors version
 pub const ADDRESS: Address = Address::new_id(4);
+/// Select address based on actors version
+fn _select_address_based_on_actors_version() -> Address {
+    todo!()
+}
```

Comment:
A TODO comment is more appropriate than an unused function, here. Just make it a dev comment rather than a documentation comment.